masterlist.yaml

```diff
@@ -5172,6 +5172,9 @@ plugins:
 
   - name: 'DIVERSE SKYRIM.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/7707/' ]
+    after:
+      - 'Kireina SSE.esl'
```

Comment:
This after shouldn't be required as `DIVERSE SKYRIM.esp` will sort after the `.esl` version.
Maybe if they ESM flagged `DIVERSE SKYRIM.esp`, can't imagine why they would though.
It's not going to cause any issue either way so it can be left in the pr.